static/styles/audit-report/audit.css

```diff
@@ -95,13 +95,31 @@ a:hover {
   color: black;
 }
 
+#searchInput {
+  margin-bottom: 20px;
+  width: 100%;
+}
+
 #resultTable {
-  display: table;
-  width: 76vw;
+  border-spacing: 0.8em;
+}
+
+.resultContainer tbody:before {
+  content: "@";
+  display: block;
+  line-height: 10px;
+  text-indent: -99999px;
```

Comment:
Seems a bit unusual

Reply:
It's just for spacing between thead and body, more like a hack